src/Spd.Manager.Licence/BizProfileValidation.cs

```diff
@@ -21,6 +22,17 @@ public BizProfileUpdateRequestValidator()
         RuleFor(r => r.SoleProprietorSwlPhoneNumber)
             .NotEmpty()
             .When(r => r.BizTypeCode == BizTypeCode.NonRegisteredSoleProprietor || r.BizTypeCode == BizTypeCode.RegisteredSoleProprietor);
-        //todo: add rule for branches
+        RuleFor(r => r.Branches)
+            .ForEach(r => r
+                .Must(r => r.BranchAddress?.AddressLine1.IsNullOrEmpty() != true)
```

Comment:
do not use it if it is from Token.
Just use string.IsNullOrWhiteSpace.

Reply:
Fixed.